src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml

```
@@ -56,7 +56,6 @@ sylius_fixtures:
                             WORLD:
                                 name: 'World'
```

You should change the name of zone:
```suggestion
                                name: 'Rest of the World'
```